server.go

```diff
@@ -40,8 +42,24 @@ func main() {
 	icd10URL := flag.String("icd10URL", "https://www.cms.gov/Medicare/Coding/ICD10/Downloads/2016-Code-Descriptions-in-Tabular-Order.zip", "url for icd-10 code definition zip")
 	subscriptionFlag := flag.Bool("subscriptions", false, "enables limited support for resource subscriptions (default: false)")
 	reqLog := flag.Bool("reqlog", false, "Enables request logging -- do NOT use in production")
+	logFileFlag := flag.String("logdir", "", "Path to a directory for ie and gin logs to be written to.")
 	flag.Parse()
 
+	lfpath := getConfigValue(logFileFlag, "IE_LOG_DIR", "")
+	if lfpath != "" {
+		err := os.Mkdir(lfpath, 0755)
+		if err != nil && !os.IsExist(err){
+			fmt.Println("Error creating log directory:" + err.Error())
+		}
+
+		lf, err := os.OpenFile(lfpath + "/ie.log", os.O_RDWR|os.O_APPEND, 0755)
+		if os.IsNotExist(err) {
+			lf, err = os.Create(lfpath + "/ie.log")
+		} else if err != nil {
```

Comment:
This needs to be a new `if` statement instead of an `if else`.  Otherwise the `else` (`log.SetOutput(lf)`) won't get called when `os.IsNotExist` (since it's all part of the same big if branch).